cordapp/README.md

```diff
@@ -50,31 +61,31 @@ do some manual work to set up a docker container (instructions below).
    ```
    apt-get -y update && apt-get install -y openjdk-8-jdk-headless && export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
    ```
-5. Change directory to your project and run the project as you would under Linux:
+5. Change directory to your project and run the project.
     ```
     cd <project-directory>
     ./gradlew workflows:test
     ```
 
-Alternatively, ubuntu 20.04 via WSL 2 ([Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install)
+Alternatively, Ubuntu 20.04 via WSL 2 ([Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install)
 may also prove to work for you, though this has not been extensively tested.
 
 For an explanation of the Docker command used above, see
 [the tutorials](https://docs.conclave.net/running-hello-world.html#appendix-summary-of-docker-command-options).
 
 ## Corda Node Identity Validation
-The [certificates](certificates) folder contains the truststore.jks Java KeyStore that contains the Corda Root
-certificate authority (CA) public key that can be used for development or testing purposes. It is used to generate 
-the trustedroot.cer that is then embedded as a resource in the enclave, and used to validate a Corda node's identity 
-when a message is relayed from the host to the enclave.
+The [certificates](certificates) folder contains the truststore.jks Java KeyStore that has the Corda Root
+Certificate Authority (CA) public key. This public key can be used for development or testing purposes. It creates
```

Comment:
It is not obvious here what creates `trustedroot.cer`. I would keep the old wording that the key is used to generate the trustedroot.cer certificate.

Reply:
I have reworded the sentence again as I feel that it wasn't clear in the original document too that the key was used to generate the certificate.

Reply:
You do mention in your comment that 'key is used to generate the certificate', but in the text it says 'the public key also creates the certificate' which are the words I don't prefer. Because key itself can't create the certificate. But it can be used to do it.

Reply:
I have made the change.